index.js

```diff
@@ -1,17 +1,61 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function (a, b) {
+      return a - b;
+    });
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos <= this.items.length && pos >= 0) {
+      var value = this.items[pos];
+      return value;
+    } else {
+      throw new Error('OutofBounds');
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.items.length > 0) {
```

Comment:
You could have two other aproaches here:
1) Sort everthing and return the first or last value depending if you sorte ascending or descending.
2) Math.max(), take a look a this method.